packages/bot-web-ui/src/components/dashboard/dashboard-component/constants.ts

```diff
@@ -5,7 +5,7 @@ export type TSidebarItem = {
     content: string[];
 };
 
-export const SIDEBAR_INTRO: TSidebarItem[] = [
+export const sidebar_intro: TSidebarItem[] = [
```

Comment:
We use `UPPER_SNAKE_CASE` for constants
`SIDEBAR_INTRO` was fine.
[Our naming convention](https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#naming-conventions)

Reply:
reverted the changes